.buildkite/pipeline.yml

```diff
@@ -1,9 +1,16 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
+---
+
+agents:
+  queue: mac
+
+env:
+  IMAGE_ID: xcode-16.4
```

Comment:
I was wary of bumping from Xcode 15.2 to 16.4 without double-checking nothing broke… but CI seems to be green on the podspec validation, so I guess it's ok…